service/service.go

```diff
@@ -87,6 +95,87 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var newOperatorBackOff *backoff.ExponentialBackOff
+	{
+		newOperatorBackOff = backoff.NewExponentialBackOff()
+		newOperatorBackOff.MaxElapsedTime = config.Viper.GetDuration(config.Flag.Service.Operator.OperatorBackOffDuration)
+	}
+
+	var newServiceResource framework.Resource
+	{
+		serviceConfig := serviceresource.DefaultConfig()
+
+		newServiceResource, err = serviceresource.New(serviceConfig)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	var resources []framework.Resource
+	{
+		resources = []framework.Resource{
+			newServiceResource,
+		}
+
+		logWrapConfig := logresource.DefaultWrapConfig()
+		logWrapConfig.Logger = config.Logger
+		resources, err = logresource.Wrap(resources, logWrapConfig)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		retryWrapConfig := retryresource.DefaultWrapConfig()
+		retryWrapConfig.BackOffFactory = func() backoff.BackOff {
+			return backoff.WithMaxTries(backoff.NewExponentialBackOff(), uint64(config.Viper.GetDuration(config.Flag.Service.Resource.Retries)))
```

Comment:
I am unsure again if this is the correct place to get the info out of `config`

Reply:
1. `Viper.GetDuration` does not make sense here. We probably want to do [`Viper.GetInt`](https://godoc.org/github.com/spf13/viper#GetInt) and do the casting then.
2. When we use the information provided by the configured viper we should make sure it fits our requirements. I can see somebody fooling around with the config map and we get 0 here. This should be invalid. This also shows that configuring this is probably not necessary at all even though other people did it already in other places.

Reply:
Thanks for the explanation. I thought that operatorkit maybe does some validation, but yeah validating it here would make more sense.
I moved this to fixed values for now anyway, to remove all those unneeded flags.

Reply:
The BackOff of the `backoff` package gets configured here, nothing from operatorkit.